src/apps.json

```diff
@@ -2495,10 +2495,11 @@
     },
     {
       "name": "Metacast",
-      "pattern": "^Metacast(/.*)?$",
+      "pattern": "^Metacast/",
       "examples": [
-        "Metacast/189 CFNetwork/1485 Darwin/23.1.0",
-        "Metacast"
+        "Metacast/0.73.5 (ios)",
+        "Metacast/1.2.0 (android)",
+        "Metacast/1.0 WebApp +https://metacast.app"
```

Comment:
From @johnspurlock via Twitter about the "WebApp" example:

> ... since that doesn't hit the media files (yet?) can you remove from the pr?  all of the UAs are meant to represent "listener-initiated playback" (as much as possible with the world as it is)

Will remove. We will add this when we have a web based player built into the web app.